src/examples/learn_quant/grammar.py

```diff
@@ -0,0 +1,49 @@
+from ultk.language.grammar import Grammar, Rule
+
+class QuantifierGrammar(Grammar):
```

Comment:
Can you say more about why this class is needed?  As far as I can see, you're specifying the `index` rules in `grammar.yml`, and not actually using the methods of the class